internal/states/statemgr/filesystem.go

```diff
@@ -92,6 +92,16 @@ func NewFilesystemBetweenPaths(readPath, writePath string) *Filesystem {
 	}
 }
 
+// SaveErroredTestStateFile saves the state present in memory as a stateFile in disk
+// and is invoked when "tofu test" fails to destroy resources.
+func SaveErroredTestStateFile(stateFilePath string, state *states.State) error {
```

Comment:
I don't think we need to pass in the stateFilePath as it never changes. Hardcoding the value as it's passed into the NewFilesystem call is fine.

Reply:
done